src/modules/AppComponents/displayComments.js

```diff
@@ -0,0 +1,27 @@
+import { getComments } from '../InvolvementService/commentService.js';
+
+const generateComment = (comment) => `
+    <article class="comment">
+      <p><strong>${comment.username}</strong> - ${comment.creation_date}</p>
+      <p>${comment.comment}</p>
+    </article>
+  `;
+
+const renderComments = async (id, commentPopupEl) => {
+  const comments = await getComments(id);
+
+  const commentsContainerEl = commentPopupEl.querySelector('.comment_cards__container');
+  const commentSectionEl = commentPopupEl.querySelector('.movie__comments');
+
+  commentSectionEl.classList.toggle('fullwidth');
+  commentsContainerEl.innerHTML = '';
+  const commentsTitle = commentPopupEl.querySelector('h3');
+
+  commentsTitle.textContent = `Comments (${comments.length})`;
+
+  comments.forEach((comment) => {
+    commentsContainerEl.insertAdjacentHTML('beforeend', generateComment(comment));
+  });
+};
```

Comment:
1.  I would suggest that you load the comments progressively instead of loading them all at once ;)
2. When there's no comment for a defined series, kindly display such a message to the user for a better experience **"There's not yet comment"**.